wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack+Backup.swift

```diff
@@ -331,7 +331,7 @@ public extension CoreDataStack {
         try coordinator.remove(store)
     }
 
-    private static func prepareStoreForBackupImport(
+    private static func prepareStoreForBackupImport( // TODO: relevant for import use case?
```

Comment:
yes it's relevant, but I guess it means that we also should retrieve the values from current db, and set them to the imported backup. See:

```
    func prepareToImportBackup() {
        setPersistentStoreMetadata(nil as Data?, key: ZMPersistedClientIdKey)
        setPersistentStoreMetadata(nil as Data?, key: PersistentMetadataKey.pushToken.rawValue)
        setPersistentStoreMetadata(nil as Data?, key: PersistentMetadataKey.pushKitToken.rawValue)
        setPersistentStoreMetadata(nil as Data?, key: PersistentMetadataKey.lastUpdateEventID.rawValue)
        _ = makeMetadataPersistent()
    }
    ```